internal/scheduler/scheduling/scheduling_algo_test.go

```diff
@@ -399,11 +399,12 @@ func TestSchedule(t *testing.T) {
 			schedulingConfig: testfixtures.TestSchedulingConfig(),
 			executors:        []*schedulerobjects.Executor{test1Node32CoreExecutor("executor1")},
 			queues:           []*api.Queue{{Name: "queue1", PriorityFactor: 0.01}, {Name: "queue2", PriorityFactor: 0.01}},
-			queuedJobs:       testfixtures.N32Cpu256GiJobs("queue2", testfixtures.PriorityClass0, 1),
+			queuedJobs:       testfixtures.N16Cpu128GiJobs("queue2", testfixtures.PriorityClass0, 1),
```

Comment:
I believe this unit test was asserting the wrong thing

It was showing a queue above fairshare would kick a queue below fairshare out of the way - which isn't how it should work

I've inverted it, so the preempting queue is at fairshare and the preempted queue is above fairshare, so it now asserts the expected behaviour